core/web/resolver/query.go

```diff
@@ -153,7 +154,11 @@ func (r *Resolver) Job(ctx context.Context, args struct{ ID graphql.ID }) (*JobP
 	if err != nil {
 		if errors.Is(err, sql.ErrNoRows) {
 			return NewJobPayload(r.App, nil, err), nil
+		}
 
+		//We still need to show the job in UI/CLI even if the chain id is disabled
+		if strings.Contains(err.Error(), "failed to get chain with id") {
```

Comment:
Is there a way to use `errors.Is(err,` here intead?

Reply:
Changed to `ErrNoSuchChainID` in [`c3540b5` (#9948)](https://github.com/smartcontractkit/chainlink/pull/9948/commits/c3540b5e57e98fcc2a68083e22424838d73c328c)